ttt-ttpe/src/main/java/com/skynav/ttpe/text/BreakPhrase.java

```diff
@@ -54,6 +56,14 @@ public boolean isParagraphBreak() {
         return type == Break.PARAGRAPH;
     }
 
+    private static List<StyleAttributeInterval>  getBreakAttributes()
+    {
+        // Don't collapse the break text using default whitespace rules.
+        List<StyleAttributeInterval> attributes = new java.util.ArrayList<StyleAttributeInterval>();
+        attributes.add(new StyleAttributeInterval(StyleAttribute.WHITESPACE, Whitespace.PRESERVE, 0, 1));
```

Comment:
When a file has multiple `<br>` elements in a row, these get translated into multiple subsequent line separator characters (via `getBreakText` above.) We then force `xml:whitespace` to prevent them from being consolidated (changes in `LineLayout.java`.)

I found an additional unit test in our commit history, `ttml2-break-break.xml`, that would be a good idea to include in this PR. Adding it shortly.